src/frontend.cpp

```diff
@@ -921,8 +921,20 @@ static bool startAudioAndZoomOptionsMenu()
 	addMultiBut(psWScreen, FRONTEND_BOTFORM, FRONTEND_QUIT, 10, 10, 30, 29, P_("menu", "Return"), IMAGE_RETURN, IMAGE_RETURN_HI, IMAGE_RETURN_HI);
 
 	//add some text down the side of the form
+	// TRANSLATORS: "AUDIO" options determine the volume of game sounds.
+	// "OPTIONS" means "SETTINGS".
 	addSideText(FRONTEND_SIDETEXT, FRONTEND_SIDEX, FRONTEND_SIDEY, _("AUDIO / ZOOM OPTIONS"));
-
+	const char *audioAndZoomOptionsExtraString = \
+		// no empty string is used because msgmerge(1) will otherwise
+		// initialize its msgstr to a copy of header contents
+		// TRANSLATORS: This placeholder message provides an extra line
+		// for the translation of the sidetext "AUDIO / ZOOM OPTIONS"
+		_(" ");
```

Comment:
Since this is a placeholder, why don't we change it to something obvious like `"<AUDIO / ZOOM OPTIONS - OPTIONAL SECOND LINE - SET TO EMPTY IF NOT REQUIRED>"`, and then just ignore it if it's empty or that string.

Reply:
> Since this is a placeholder, why don't we change it to something obvious like
> `"<AUDIO / ZOOM OPTIONS - OPTIONAL SECOND LINE - SET TO EMPTY IF NOT
> REQUIRED>"` [...]

This is too obscure.
My translator comment should sufficiently explain the message's meaning.
Alternatively, a msgctxt keyword could be used, but it is unnecessary.

> [...] and then just ignore it if it's empty or that string.

I have implemented this.
In addition, removing the message " " from all PO files ensures that
`update-po.sh; make update-po` will recreate it as untranslated message
directly below the message which it complements:

```
#. TRANSLATORS: "AUDIO" options determine the volume of game sounds.
#. "OPTIONS" means "SETTINGS".
#: src/frontend.cpp:926
msgid "AUDIO / ZOOM OPTIONS"
msgstr "AUDIO- UND ZOOMEINSTELLUNGEN"

#. TRANSLATORS: This placeholder message provides an extra line
#. for the translation of the sidetext "AUDIO / ZOOM OPTIONS"
#: src/frontend.cpp:932
msgid " "
msgstr ""
```

Reply:
Wouldn't it make more sense to either allow a `\n` in the msgstr or implement word wrapping than to have a separate one for the second line?

Reply:
> Wouldn't it make more sense to either allow a `\n` in the msgstr [...]

I briefly considered this, but there were too many problems.
You'd have to parse the msgstr to ensure all escape sequences except for
the first newline are ignored, warn about any syntax errors and explain
the format to translators, who may be unfamiliar with format specifiers.

> or implement word wrapping [...]

Line breaking rules differ across languages ([example](https://en.wikipedia.org/wiki/Line_breaking_rules_in_East_Asian_languages)).
Supporting non-breaking (hard) and optional (soft) hyphens is required
to support merely the conventions of English. I prefer not to.

Of course, you can prove me wrong with a patch implementing your ideas.